.github/workflows/comment_test.yml

```diff
@@ -41,5 +42,6 @@ jobs:
       - name: Infracost comment
         uses: ./comment
         with:
-          path: '["./testdata/comment/simple_breakdown.json", "./testdata/comment/simple_breakdown.json"]'
+          path: "./testdata/comment/*.json"
           target-type: commit
+          tag: "infracost-commit"
```

Comment:
One way to fix the CI issue is to set `behavior: new` here. `dry-run` flag doesn't post the comment, but it still makes an API request to fetch the comments with default `update` behavior. Behavior `new` won't do it. I think we should update `dry-run` to do any external API requests, but it's out of scope of this PR.

Reply:
I've updated this to `new` in https://github.com/infracost/actions/pull/57/commits/466dd8e10f13ebd31a91e3907b9a5a6a60217ac3. But this doesn't perform a dry run here  it actually posts the comment (see bottom of pr)

Reply:
Ah, my bad, mixed it up with the examples. 👍